simplesource-command-serialization/src/main/java/io/simplesource/kafka/serialization/avro/AvroSerdes.java

```diff
@@ -107,36 +107,30 @@ private static Schema commandResponseKeySchema() {
         private static final String REASON = "reason";
         private static final String ADDITIONAL_REASONS = "additionalReasons";
         private static final String ERROR_MESSAGE = "errorMessage";
-        private static final String ERROR_CODE = "errorCode";
+        private static final String ERROR_CLASS = "errorClass";
+        private static final String ERROR = "error";
         private static final String WRITE_SEQUENCE = "writeSequence";
         private static final String AGGREGATION = "aggregate_update";
 
         private static GenericRecord fromReason(final Schema schema, final CommandError commandError) {
             return new GenericRecordBuilder(schema)
                     .set(ERROR_MESSAGE, commandError.getMessage())
-                    .set(ERROR_CODE, commandError.getReason().name())
+                    .set(ERROR_CLASS, commandError.getClass().getCanonicalName())
```

Comment:
`ERROR_MESSAGE ` and `ERROR_CLASS` are here for debug purpose.

Reply:
Because we are embedding Java serialised classes in Avro, we won't otherwise be able to view the exception on via the avro console viewer, so we probably still need these, much as I'd like to remove them.